src/components/Note/Note.js

```diff
@@ -85,6 +88,8 @@ const Note = ({ annotation }) => {
       core.deselectAllAnnotations();
       core.selectAnnotation(annotation);
       core.jumpToAnnotation(annotation);
+      // Need this delay to ensure all other event listeners fire before we open the line
+      setTimeout(() => dispatch(actions.openElement('annotationLineConnector'), 300));
```

Comment:
This was interesting. The timing of the events was causing the line to never open, as the `core.jumpToAnnotation` would trigger the onscroll event (effectively closing the line), but it was the last call, so after line 92 executed.